src/pages/components/AddAccountForm/index.tsx

```diff
@@ -0,0 +1,239 @@
+import { Avatar, Box, Button, CircularProgress, Dialog, DialogActions, DialogContent, DialogTitle, IconButton, Stack, styled, TextField, Tooltip, Typography } from "@mui/material"
+import { useEffect, useState } from "react";
+import { toast } from "react-toastify";
+import AddIcon from '@mui/icons-material/Add';
+import HighlightOffTwoToneIcon from '@mui/icons-material/HighlightOffTwoTone';
+import { lightGreen, lightBlue, red, grey } from "@mui/material/colors";
```

Comment:
Let's try to use all colors from the theme file. It will make it easier once we decide to implement a dark mode or change the app style in the future. You can add any color that you need there.